waveform_editor/gui/plotter_view.py

```diff
@@ -46,7 +48,9 @@ def plot_waveform(self, waveform):
         xlabel = "Time (s)"
         ylabel = "Value"
 
-        if waveform is None or not waveform.tendencies:
+        if waveform is None or (
+            isinstance(waveform, Waveform) and not waveform.tendencies
+        ):
             return hv.Curve(([], []), xlabel, ylabel)
         times, values = waveform.get_value()
```

Comment:
Perhaps we could instead update the get_value() logic in the Waveform class to return `([], [])` when it has no tendencies and remove the isinstance and waveform.tendencies check here?